virus/hespivirus_rivalense.md

```diff
@@ -0,0 +1,273 @@
+a 41 nt suffix of non-repeating bases. To check for potential assembly errors, we realign the reads from SRR17250291 to
+our contig using bowtie2 [19], resulting in a mean 1026x coverage. We observe high coverage (~300x) of the bases
+following the poly(A) stretch, which decays to 1x over the next 20 bases. However, there is a coverage bias of ends of
+contigs due to the assembly process, and a similar decay is observed in the 5’ end. Hence, our analysis remains
+inconclusive.
+
+We find 10 ORFs in our viral contig with ORFfinder and restricting to 75 nt non-nested ORFs. InterproScan [20] predicts
+domains in 3 of 10 ORFs, ignoring disorder predictions, coils, and SignalP-noTM, which is not annotated further but
+seems to indicate a signal peptide. The three ORFs lie on the positive strand, so we will name them in downstream order:
+ORF1 (522-9416 nt), ORF2 (9417-9509 nt), and ORF3 (9906-10388 nt). These ORFs are predicted to have the following
+domains:
+
+- An *Alphavirus*-like methyltransferase (MTase) domain (IPR002588) in ORF1.
+- An rRNA MTase FtsJ domain (IPR002877) in ORF1.
+- An alpha-ketoglutarate-dependent dioxygenase AlkB-like domain (IPR027450) in ORF1.
+- A RNA virus helicase core domain (IPR027351) in ORF1.
+- An RdRp from *Alsuviricetes* (IPR001788) in ORF1.
+- A transmembrane helix (TMh) that is part of a membrane-bound protein in ORF2.
+- A Tobacco mosaic virus-like coat protein (TMVC) (IPR001337) in ORF3.
+
+We are unable to find any elements encoding for non-coding RNAs in `Hriv_contig`, using Infernal cmscan 1.1.2. The
+following figure summarizes the contig-wide coverage, ORFs, predicted domains, and RdRp A, B, C motifs.
+
+![](imgs/hespivirus_rivalense/contig.png)
+
+### AlphaFold2: RdRp structure
+
+We use the putative RdRp sequence above and ColabFold [21] with its default settings. We depict one seed structure with
+Py3DMol [22].
+
+<div align="center">
+<img src="imgs/hespivirus_rivalense/rdrp_alphafold.png" alt="drawing" width="350"/>
+</div>
+
+### Phylogenetic analysis
+
+We use the putative RdRp sequence from above. For BLASTP [23], we search against the nr database restricted to RNA
+viruses (taxid:2559587), and we set the maximum number target sequences to 1000. Filtering the candidates with >80%
+query coverage and >40% identity leaves 191 sequences (including the *H. rivalense* RdRp). We further remove entries
+with duplicate scientific names or named "ssRNA positive-strand virus sp." and "Virgaviridae sp.", tie-breaking by E
+score. On the remaining 81 sequences, we perform a multiple sequence alignment (MSA) using MUSCLE [24] and visualize the
+result below using Jalview [25]. For visual clarity, we have trimmed the MSA to the *H. rivalense* RdRp palmprint and
+included only the top 19 BLASTP hits. The A, B, and C motifs of the *H. rivalense* palmprint are annotated in the last
+row, and by inspection, we can see that they are well-conserved.
+
+![](imgs/hespivirus_rivalense/msa.png)
```

Comment:
Please include the FASTA MSA data file and .tree as supporting information for this figure. You can store it in the `imgs/hespivirus_rivalense/` folder along with image data.